recipe/build.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
+ if [ $(uname) == Darwin ]; then
```

Comment:
This is a really bad thing to put in a build script - what if somebody else wants to build it? :-1: 

Instead, this should live in the **CI scripts**. You could test that out here, then make the change in conda-smithy.